snow/consensus/snowman/consensus_test.go

```diff
@@ -260,11 +259,9 @@ func AddToUnknownTest(t *testing.T, factory Factory) {
 		HeightV: snowmantest.GenesisHeight + 2,
 	}
 
-	// Adding a block with an unknown parent means the parent must have already
-	// been rejected. Therefore the block should be immediately rejected
-	require.NoError(sm.Add(context.Background(), block))
-	require.Equal(snowmantest.GenesisID, sm.Preference())
-	require.Equal(choices.Rejected, block.Status())
```

Comment:
Why is the preference check removed? I assume the rejected check is removed because consensus now only returns an error, but does not mark the failed to add block as rejected.

Reply:
I think we don't have to since we don't add to consensus anymore. Previously we were adding and immediately deciding rejected, now we're just dropping it from consensus entirely and not making a decision at all.

Reply:
Previously, this was testing that the preference stayed the same as it was previously (ie. preference is still genesis after adding the new block fails) though since it dropped and rejected the block in the same place.

I agree we don't need the check in the test though.

Reply:
It just didn't seem like a useful check anymore. I can revert that if we feel like it was important to test.